packages/core/src/codewhisperer/client/user-service-2.json

```diff
@@ -18,13 +18,25 @@
                 "method": "POST",
```

Comment:
Looks like formatting changes were made to this file. The service models have differently-formatted versions, please ensure that you didn't use a version with different formatting, to avoid churn in the git history.

Reply:
Checked with some others on this, and apparently they're not able to provide me with a version that has formatting consistent with how this file currently looks. I suppose I could manually fix it or add just the changes needed for `/transform` but that doesn't seem like a great idea

Reply:
Looks like someone else was able to find the right one, updated